terraform/vpc.tf

```diff
@@ -90,3 +90,177 @@ resource "aws_route_table_association" "association" {
   subnet_id = "${aws_subnet.build_public_subnet.id}"
   route_table_id = "${aws_route_table.build_public_route_table.id}"
 }
+
+# ACL for the private subnet of the VPC
+resource "aws_network_acl" "build_default_acl" {
+  vpc_id = "${aws_vpc.build_vpc.id}"
+  subnet_ids = [
+    "${aws_subnet.build_private_subnet.id}"
+  ]
+
+  # Allow ephemeral ports from anywhere
+  ingress {
+    protocol = "tcp"
+    rule_no = 100
+    action = "allow"
+    cidr_block = "0.0.0.0/0"
+    from_port = 1024
+    to_port = 65535
+  }
+
+  # Allow HTTP (needed for apt-get)
+  egress {
+    protocol = "tcp"
+    rule_no = 100
+    action = "allow"
+    cidr_block = "0.0.0.0/0"
+    from_port = 80
+    to_port = 80
+  }
+
+  # Allow HTTPS
+  egress {
+    protocol = "tcp"
+    rule_no = 110
+    action = "allow"
+    cidr_block = "0.0.0.0/0"
+    from_port = 443
+    to_port = 443
+  }
+
+  tags = "${var.tags}"
+}
+
+# ACL for the public-facing subnet of the VPC
+resource "aws_network_acl" "build_public_acl" {
+  vpc_id = "${aws_vpc.build_vpc.id}"
+  subnet_ids = [
+    "${aws_subnet.build_public_subnet.id}"
+  ]
+
+  # Allow ephemeral ports from anywhere
+  ingress {
+    protocol = "tcp"
+    rule_no = 100
+    action = "allow"
+    cidr_block = "0.0.0.0/0"
+    from_port = 0
```

Comment:
@jsf9k shouldn't this be from_port = 1024 ?

Reply:
Yes.  Good catch!